package.json

```diff
@@ -78,7 +79,7 @@
   },
   "dependencies": {
     "axios": "^0.16.1",
-    "cli-table": "^0.3.1",
+    "cli-table2": "^0.2.0",
```

Comment:
Is the dependency-name really `cli-table2`. This looks odd to me :)

Reply:
It is.  :)  It's a rewrite of `cli-table` with some level of API compatibility.